monocle/worker.py

```diff
@@ -830,8 +830,29 @@ def travel_speed(self, point):
                     if fort.id not in FORT_CACHE.pokestops:
                         pokestop = self.normalize_pokestop(fort)
                         db_proc.add(pokestop)
-                elif fort not in FORT_CACHE:
-                    db_proc.add(self.normalize_gym(fort))
+                else:
+                    if fort not in FORT_CACHE:
+                        db_proc.add(self.normalize_gym(fort))
+                    if fort.HasField('raid_info'):
+                        fort_raid = {}
+                        fort_raid['external_id'] = fort.id
+                        fort_raid['raid_battle_ms'] = fort.raid_info.raid_battle_ms
+                        fort_raid['raid_spawn_ms'] = fort.raid_info.raid_spawn_ms
+                        fort_raid['raid_end_ms'] = fort.raid_info.raid_end_ms
+                        fort_raid['raid_level'] = fort.raid_info.raid_level
+                        fort_raid['complete'] = fort.raid_info.complete
+                        fort_raid['pokemon_id'] = ""
+                        fort_raid['cp'] = ""
+                        fort_raid['move_1'] = ""
+                        fort_raid['move_2'] = ""
+                        if fort.raid_info.HasField('raid_pokemon'):
+                            fort_raid['pokemon_id'] = fort.raid_info.raid_pokemon.pokemon_id
+                            fort_raid['cp'] = fort.raid_info.raid_pokemon.cp
+                            fort_raid['move_1'] = fort.raid_info.raid_pokemon.move_1
+                            fort_raid['move_2'] = fort.raid_info.raid_pokemon.move_2
```

Comment:
these lines (after `if fort.HasField('raid_info'):`) should be part of `normalize_raid`